cmd/fleetctl/scripts_test.go

```diff
@@ -20,8 +19,6 @@ func TestRunScriptCommand(t *testing.T) {
 		License: &fleet.LicenseInfo{
 			Tier: fleet.TierPremium,
 		},
-		// increase the default timeout to 90 seconds to match the production server
-		HTTPServerConfig: &http.Server{WriteTimeout: 90 * time.Second}, // nolint:gosec
```

Comment:
For some reason, setting this timeout was causing an issue with fetching the license information at runtime. In spite of the license being set, when the test server would pull the license during endpoint handling, the license would come back as nil. Not really sure why this is the case, but removing it allowed my new test to pass as well as all existing tests, so went ahead and removed it.

Reply:
Please try with:
```go
&service.TestServerOpts{
  License: &fleet.LicenseInfo{
    Tier: fleet.TierPremium,
  },
},
&service.TestServerOpts{
  HTTPServerConfig: &http.Server{WriteTimeout: 90 * time.Second}, // nolint:gosec
},
```

Reply:
Though I don't really know what's going on. It seems we don't support setting multiple options?

Reply:
hmm yeah that worked... /shrug